wordpress-mobile/WordPress-iOS
# File Block Media Source label improvement

## Description

This change simply updates the label on the File Block's `media-upload` options sheet. Previously the label was `Choose from device` but as stated by @ceyhun in the [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/15839#issuecomment-778340310) this label is used to open the Media Picker UI in other blocks, eg. Image block. 

This change aligns the behavior of this block with the other media blocks by using the `Other apps` label that represents the action being committed. 

## Testing 

1. Create a new post. 
2. Add a File Block. 
3. Notice that the verbiage is now `Other apps` instead of `Choose from device.` when the media options are shown.

Before | After 
--------|-------
  <kbd><img src="https://user-images.githubusercontent.com/1509205/118917495-a256df80-b8f6-11eb-93aa-a36d7e68fd83.PNG" width="320"></kbd>   |  <kbd><img src="https://user-images.githubusercontent.com/1509205/118917479-9a973b00-b8f6-11eb-9e2c-168984524bc1.png" width="320"></kbd>

## Regression Notes

1. Potential unintended areas of impact

There are no areas of impact based on my understanding of this change. A string is being removed and another string is being utilized in multiple places.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
I relied on the smoke testing I did of the app with this PR. 

3. What automated tests I added (or what prevented me from doing so)
I ran the optional tests to verify they will pass.
